Yosemite/YosemiteTests/Tools/Media/Media+Equatable.swift

```diff
@@ -1,18 +1,22 @@
 @testable import Yosemite
 
-// MARK: - Equatable Conformance
+// MARK: - Equatable Override
 //
-extension Media: Equatable {
-    public static func == (lhs: Media, rhs: Media) -> Bool {
-        return lhs.mediaID == rhs.mediaID &&
-            lhs.date == rhs.date &&
-            lhs.fileExtension == rhs.fileExtension &&
-            lhs.mimeType == rhs.mimeType &&
-            lhs.src == rhs.src &&
-            lhs.thumbnailURL == rhs.thumbnailURL &&
-            lhs.name == rhs.name &&
-            lhs.alt == rhs.alt &&
-            lhs.height == rhs.height &&
-            lhs.width == rhs.width
+extension Media {
+    public override func isEqual(_ object: Any?) -> Bool {
+        guard let object = object as? Media else {
+            return false
+        }
+
+        return mediaID == object.mediaID &&
+        date == object.date &&
+        fileExtension == object.fileExtension &&
+        mimeType == object.mimeType &&
+        src == object.src &&
+        thumbnailURL == object.thumbnailURL &&
+        name == object.name &&
+        alt == object.alt &&
+        height == object.height &&
+        width == object.width
```

Comment:
This is the right indentation 👍 
```suggestion
            date == object.date &&
            fileExtension == object.fileExtension &&
            mimeType == object.mimeType &&
            src == object.src &&
            thumbnailURL == object.thumbnailURL &&
            name == object.name &&
            alt == object.alt &&
            height == object.height &&
            width == object.width
```